integr8ly/integreatly-operator
# Intly 9431 watch namespace for cidr range

# Description
https://issues.redhat.com/browse/INTLY-9431
Add the ability to watch a namespace for a cidr range and apply it to the cloud-resources-aws-strategies configmap 

## Type of change

- [X ] New feature (non-breaking change which adds functionality)

## Validation
If running locally:
1. remove CIDR range from deploy/cro-aws-config.yml
2. run `make cluster/prepare/project`
3. Apply label to namespace `oc label namespace redhat-rhmi-operator cidr=10.1.0.0-23` **-** is used because **/** is an invalid character in the label value. 
4. run `make cluster/prepare/local` and verify the configmap gets updated with the cidr value "10.1.0.0/23" and that rhmi installs.

example configmap attached to compare against. 

[configmap-cloud-resources-aws-strategies.txt](https://github.com/integr8ly/integreatly-operator/files/5051953/configmap-cloud-resources-aws-strategies.txt)
